libexec/scoop-update.ps1

```diff
@@ -123,32 +123,42 @@ function update_scoop() {
         }
     }
 
-    if ((Get-LocalBucket) -notcontains 'main') {
-        info "The main bucket of Scoop has been separated to 'https://github.com/ScoopInstaller/Main'"
-        info "Adding main bucket..."
-        add_bucket 'main'
-    }
+    # This should have been deprecated after 2019-05-12
+    # if ((Get-LocalBucket) -notcontains 'main') {
+    #     info "The main bucket of Scoop has been separated to 'https://github.com/ScoopInstaller/Main'"
+    #     info "Adding main bucket..."
+    #     add_bucket 'main'
+    # }
 
     ensure_scoop_in_path
     shim "$currentdir\bin\scoop.ps1" $false
 
-    Get-LocalBucket | ForEach-Object {
-        write-host "Updating '$_' bucket..."
+    foreach ($bucket in Get-LocalBucket) {
+        Write-Host "Updating '$bucket' bucket..." -NoNewline
+
+        $bucketLoc = Find-BucketDirectory $bucket -Root
 
-        $loc = Find-BucketDirectory $_ -Root
-        # Make sure main bucket, which was downloaded as zip, will be properly "converted" into git
-        if (($_ -eq 'main') -and !(Test-Path "$loc\.git")) {
-            rm_bucket 'main'
-            add_bucket 'main'
+        if (!(Test-Path (Join-Path $bucketLoc '.git'))) {
+            if ($bucket -eq 'main') {
+                # Make sure main bucket, which was downloaded as zip, will be properly "converted" into git
+                Write-Host " Converting 'main' bucket to git..." -NoNewline
+                rm_bucket 'main'
+                add_bucket 'main'
+                Write-Host ' Done.'
+            } else {
+                Write-Host ' Not a git repository. Skipped.'
+            }
+            continue
         }
 
-        Push-Location $loc
+        Push-Location $bucketLoc
         $previousCommit = (Invoke-Expression 'git rev-parse HEAD')
         git_pull -q
         if ($show_update_log) {
             Invoke-Expression "git --no-pager log --no-decorate --format='tformat: * %C(yellow)%h%Creset %<|(72,trunc)%s %C(cyan)%cr%Creset' '$previousCommit..HEAD'"
```

Comment:
Can we add something like this https://github.com/ScoopInstaller/Scoop/pull/3605/files in the end here? It will keep the update logs clean. (single line change).

Reply:
Right, change it soon.

Reply:
What will be the best **skip** word? Just like previous PR, `[scoop skip]` or others?

Reply:
Or maybe a simple `^[build]`?

Reply:
> Or maybe a simple `^[build]`?

`build` can be useful in case of Scoop core's logs. I was thinking of `(chore):` since we have used it many times - https://github.com/search?q=%28chore%29%3A+org%3AScoopInstaller&type=commits

Reply:
`* ?chore*` is used, to allow `[chore]` or `(chore)`

Reply:
You forgot to do this on line 158

Reply:
Yes, there're two lines. A force push is made to keep log simple